src/items/associated-items.md

```diff
@@ -335,6 +338,24 @@ fn main() {
 }
 ```
 
+[Constant evaluation] timing:
```

Comment:
When I immediately read this, I didn't know what it meant by "timing".  

Instead of placing this down in the "examples" section, can this code block be moved up to the paragraph that you added above?  Generally in the reference, when there is an example demonstrating some concept, we try to write that example immediately after the paragraph that introduces that concept.

Reply:
That makes sense. Moved in be494280f42e3f998309087250d71601150a036e